docs/test.ts

```diff
@@ -23,6 +23,7 @@ import {
 } from '../ext/tags'
 import { HTTP, WEB } from '../ext/types'
 import * as opentracing from 'opentracing';
+import {ClientRequest, IncomingMessage, OutgoingMessage, ServerResponse} from 'http';
```

Comment:
It looks like `ClientRequest` and `ServerResponse` are not used. And you missed spaces after `{` and before `}`
```suggestion
import { IncomingMessage, OutgoingMessage } from 'http';
```